packages/scaffold-ui/src/partials/w3m-account-wallet-features-widget/index.ts

```diff
@@ -279,6 +270,23 @@ export class W3mAccountWalletFeaturesWidget extends LitElement {
     return html`<wui-balance dollars="0" pennies="00"></wui-balance>`
   }
 
+  private tabsTemplate() {
+    const isSolana = ChainController.state.activeChain === CommonConstantsUtil.CHAIN.SOLANA
```

Comment:
Can we make it the otherway around? When Magic introduce new chains, like Bitcoin Polkadot etc. and if we include them in the future, we'll need to add more filters. Checking if EVM or not might be simpler
```suggestion
    const isEVM = ChainController.state.activeChain === CommonConstantsUtil.CHAIN.EVM
    
    if (!isEvm) { return null }
```